presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -549,7 +550,7 @@ private static Envelope getEnvelope(OGCGeometry geometry)
 
     private static void verifySameSpatialReference(OGCGeometry leftGeometry, OGCGeometry rightGeometry)
     {
-        checkArgument(leftGeometry.getEsriSpatialReference().equals(rightGeometry.getEsriSpatialReference()), "Input Geometries Spatial Reference do not match");
+        checkArgument(Objects.equals(leftGeometry.getEsriSpatialReference(), rightGeometry.getEsriSpatialReference()), "Input Geometries Spatial Reference do not match");
```

Comment:
This error message is confusing. Should "Geometries" be possessive? Also, should "References" be plural? And why is everything capitalized?

Reply:
Maybe phrase it as "Input geometries must have same spatial reference"